src/v3/plugins/github/parseReferences.js

```diff
@@ -1,19 +1,12 @@
 // @flow
 
-function findAllMatches(re: RegExp, s: string): any[] {
-  // modified from: https://stackoverflow.com/a/6323598
-  let m;
-  const matches = [];
-  do {
-    m = re.exec(s);
-    if (m) {
-      matches.push(m);
-    }
-  } while (m);
-  return matches;
-}
+export type ParsedReference = {|
+  // "@user" or "#123" or "https://github.com/owner/name/..."
+  +ref: string,
+  +refType: "basic" | "pairedWith",
```

Comment:
Might as well be `"BASIC" | "PAIRED_WITH"` for consistency.

Reply:
Done.